pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -1,23 +1,23 @@
-{ lib, stdenv, fetchFromGitHub, cmake, python3, vulkan-headers, pkg-config
-, xlibsWrapper, libxcb, libXrandr, libXext, wayland, addOpenGLRunpath }:
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, libX11, libxcb
+, libXrandr, wayland, vulkan-headers, addOpenGLRunpath }:
 
 stdenv.mkDerivation rec {
   pname = "vulkan-loader";
-  version = "1.2.162.0";
+  version = "1.2.182.0";
 
-  src = fetchFromGitHub {
-    owner = "KhronosGroup";
-    repo = "Vulkan-Loader";
-    rev = "sdk-${version}";
-    sha256 = "0w9i2pliw4ccmjyfzff4i2f3hxwsfd54jg7ahv2v634qmx59bsbi";
-  };
+  src = (assert version == vulkan-headers.version;
```

Comment:
or we just re-use the version. That would also work.

Reply:
That could work, but it would be very easy to forget to update the hash whenver `vulkan-headers` is updated. I like how this approach is more explicit.

Reply:
Exactly, I'd rather get an assert explaining exactly what's up than a hash mismatch somewhere unexpected, or worse that nix keeps using the old source because some fixed output drv with that hash already exists.

Reply:
@SuperSandro2000 what do you think?

Reply:
> that nix keeps using the old source because some fixed output drv with that hash already exists.

We could work around that by supplying the version in the name to the fetcher but we can also use the assert. Either should work.

Reply:
let's keep the assert then for the more informative error message.

would be good to get this merged!